content/mocking-mistakes.md

```diff
@@ -0,0 +1,85 @@
+Title: Mocking Mistakes
+Date: 2015-11-10
+Category: Testing
+Tags: testing, mock
+Slug: mocking-mistakes
+Author: Jon Evans
+Avatar: jonathan-evans
+
+I've discussed the value of using Python's `mock` library for testing in a couple of [previous](http://engineroom.trackmaven.com/blog/making-a-mockery-of-python/) [articles](http://engineroom.trackmaven.com/blog/real-life-mocking/). Recently, however, a [kind commenter](http://engineroom.trackmaven.com/blog/real-life-mocking/#comment-2310097361) brought to my attention an insidious error in my example code. Having looked into this sneaky mistake, I wanted to briefly discuss it - hopefully I can prevent other developers encountering similar pitfalls!
```

Comment:
Linking articles like this might no be the best? I didn't realize there were two different articles here. Plus for SEO they should be linked by title.